queries.txt

```diff
@@ -0,0 +1,71 @@
+1. select * 
+from subjects 
+order by subject asc;
+
+2. select * 
+from subjects 
+order by location asc;
+
+3. select * 
+from books 
+where title ='Little Women';
+
+4. select * 
+from books 
+where title ilike '%python%';
+
+5. select * 
+from subjects 
+where location ilike '%main st%' 
+order by subject asc; 
+
+6. select books.title from books inner join subjects on books.subject_id = subjects.id where subject ilike '%computers%';
+
```

Comment:
really good, here is an alternative - select books.title from books join subjects on books.subject_id = subjects.id where subjects.subject = 'Computers';